### ISSUE TITLE
 As a project manager I would like to check the status of the DC LinkStation A22

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/bdp-commons#440]
Like #432 
Reference DC: https://github.com/noi-techpark/bdp-commons/tree/main/data-collectors/traveltimes-a22

**Comment by dulvui** [comment]
The A22 endpoint gives httpcode "500 Internal Server Error" when fetching data from /percorrenze/anagrafica.
Authentication and de-authentication work fine.
Could it be that there are problems or changes on the endpoint /percorrenze/anagrafica?

**Comment by rcavaliere** [comment]
@dulvui thanks for the feedback, I will ask A22!

**Comment by rcavaliere** [comment]
@dulvui asked, they will check this. This should be a technical problem on their side

**Comment by rcavaliere** [comment]
@dulvui A22 should have fixed the problem. Can you check if the data is entering? If I look at this web-component I still don't see real-time data, but maybe we are still integrating the hole of historical data we miss.. 

https://webcomponents.opendatahub.testingmachine.eu/webcomponent/ffe771ee-1786-11eb-adc1-0242ac120002/fullscreen?attribs=PG5vaS1tb2JpbGl0eS10cmFmZmljIGxhbmc9ImVuIiBzdHlsZT0iLS1ub2ktZm9udC1mYW1pbHk6ICdSb2JvdG8nLCAnSGVsdmV0aWNhIE5ldWUnLCBzYW5zLXNlcmlmOyAtLW5vaS13aWR0aDogMTAwJTsgLS1ub2ktaGVpZ2h0OiAxMDAlOyAtLW5vaS1wcmltYXJ5OiAjMDA2OEI0OyAtLW5vaS1wcmltYXJ5LXJnYjogMCwgMTA0LCAxODA7IC0tbm9pLXByaW1hcnktY29udHJhc3Q6ICNmZmZmZmY7IC0tbm9pLWFjdGlvbi1yZ2I6IDg3LCAxNzMsIDU3OyAtLW5vaS1hY3Rpb246ICMzMzk5NjY7IC0tbm9pLWFjdGlvbi1jb250cmFzdDogI2ZmZmZmZiI%252BPC9ub2ktbW9iaWxpdHktdHJhZmZpYz4%253D

**Comment by dulvui** [comment]
The datacollector works now and is green in the Grafana dashboard.
But in the webcomponent the error "A22 geometry data is incomplete for given segment" is shown for most of the tracks.
Only from "Bolzano Nord to Bolzano Sud" and "Trento Nord to Trento Sud" the gemoetry data is given.
Is that correct?

**Comment by rcavaliere** [comment]
@dulvui yes it is. I have to insert manually all the geometries first...
